javascripts/discourse/components/discourse-tag-banners.js

```diff
@@ -0,0 +1,87 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+
+export default class DiscourseTagBanners extends Component {
+  @service store;
+  @service router;
+  @service site;
+  @tracked tag = null;
+  @tracked keepDuringLoadingRoute = false;
+  @tracked isIntersection = false;
+
+  get isVisible() {
+    const route = this.router?.currentRoute;
```

Comment:
can currentRoute be undefined btw? did you have cases where it was? same for name ?